tools/load-test/README.md

```diff
@@ -0,0 +1,53 @@
+# Load Test
+
+
+## Overview
+
+The purpose of the load test is to verify the right execution of the Horizonatl Pod Autoscaling in **Kyma**. It runs a **golang** app which stress a function by making thousands of http post request in set period of time defined in an environment variable. At the end of the execution of the test a notification is send to a Slack channel with the final output of the load test.
```

Comment:
There is no info that tool is used for Kubeless. I understood it only by assuming that `function` word used here to refer to Function CR. And also I saw some Kubeless env variables

Reply:
I included a reference to Kubeless here https://github.com/kyma-project/kyma/pull/2009/commits/8d218456fb22fc3c7c3eab4d58c66969a4be0d88. My belief is this tool can be extended for more areas(rather than just functions) to load test Kyma in future.